src/ui/webeiddialog.cpp

```diff
@@ -32,27 +32,22 @@
 #include <QRegularExpressionValidator>
 #include <QUrl>
 
-namespace
-{
-
-void makeLabelForegroundDarkRed(QLabel* label)
-{
-    label->setStyleSheet("color: darkred");
-}
-
-} // namespace
-
 WebEidDialog::WebEidDialog(QWidget* parent) : WebEidUI(parent), ui(new Ui::WebEidDialog)
 {
     ui->setupUi(this);
+    ui->authenticatePinLayout->setAlignment(ui->authenticationPinInput, Qt::AlignCenter);
+    ui->signingPinLayout->setAlignment(ui->signingPinInput, Qt::AlignCenter);
+    ui->authenticationPinInput->setAttribute(Qt::WA_MacShowFocusRect, false);
+    ui->signingPinInput->setAttribute(Qt::WA_MacShowFocusRect, false);
 
     okButton = ui->buttonBox->button(QDialogButtonBox::Ok);
     makeOkButtonDefaultAndconnectSignals();
 
     lineHeight = ui->authenticateOriginLabel->height();
 
-    // Hide document hash fingerprint label and PIN-related widgets by default.
-    ui->docHashFingerprintLabel->hide();
+    // Hide PIN-related widgets by default.
+    ui->authenticationPinErrorLabel->hide();
```

Comment:
If these widgets are hidden by default, they must be revealed in `onCertificateReady()` on line 150 to display retries left.